docs/userGuide/syntax/code.md

```diff
@@ -278,6 +278,10 @@ A _wrap text_ button can be added to code blocks using the `codeBlockWrapButtons
 </panel>
 <br>
 
+##### Printing optimization
+
+Markbind enhances the readability of your code blocks for printing by applying soft wrapping, ensuring code doesn't get cut off. Additionally, line numbers are added to maintain context when wrapping occurs.
```

Comment:
Sorry this is vvv anal but maybe you could use includes here! 

```suggestion
<div id="code-print-optimization">
Markbind enhances the readability of your code blocks for printing by applying soft wrapping, ensuring code doesn't get cut off. Additionally, line numbers are added to maintain context when wrapping occurs.
</div>
```

Reply:
Great advice! Sorry that I forgot this reuse feature of markbind.